script.js

```diff
@@ -78,9 +90,15 @@ function createProductListing(category) {
             p.innerHTML= product.description
             div.appendChild(p)
 
+            let h1 = document.createElement('h1')
+            productsListing.appendChild(h1)
+            h1.innerHTML= product.amount
+            div.appendChild(h1)
+
+
             let button = document.createElement('button')
             productsListing.appendChild(button)
-             button.innerHTML = 'Cart'
+             button.innerHTML = 'Add to Cart'
```

Comment:
I feel like your div.appendChild(button) should be outside of the onclick handler.  Does the button show in the UI as is?  I would assume not because it doesn't get added to the div until the onclick handler is fired.